spec/controller/completed_lessons_controller_spec.rb

```diff
@@ -1,33 +1,44 @@
 require "rails_helper"
 
 RSpec.describe CompletedLessonsController, type: :controller do
-  login_user
+
   describe 'index' do  
     it "should go to the index page" do
+      user = FactoryBot.create(:user)
+      sign_in user
       get 'index'
       expect(response).to render_template "completed_lessons/index"
     end
   end 
 
   describe 'create' do
-    it 'successfully creates a new lesson' do
+    it 'successfully creates a new completed_lesson in db' do
+      user = FactoryBot.create(:user)
+      sign_in user
       lesson = FactoryBot.create(:lesson)
-      completed_lesson = FactoryBot.create(:completed_lesson)
+      count_before = CompletedLesson.count
+      post(:create, params: {completed_lesson: FactoryBot.attributes_for(:completed_lesson, lesson_id: lesson.id)})
+      count_after = CompletedLesson.count
+      expect(count_after).to eq(count_before + 1)
       expect(CompletedLesson.last.completed).to eq(true)
-      expect(response.status).to eq(200)
+      expect(response).to redirect_to(lesson_path(lesson))
+      # expect(response.status).to eq(200)
```

Comment:
Looks like you left these comments in - did you mean to or should they be removed?

Reply:
Ah, I just noticed your question! I think that in this case, since we are testing a 'post' request (instead of a 'get') we don't necessarily return a 200 code. 

I checked in the browser when creating a new lesson, and the post request actually returns a 302 Status Code when posting a new lesson record successfully. So in this case I think we can add `expect(response.status).to eq(302)` 


<img width="589" alt="Screenshot 2020-04-23 at 17 02 19" src="https://user-images.githubusercontent.com/7221533/80149391-95957a00-8584-11ea-91ac-3e774ba4892f.png">


<img width="1430" alt="Screenshot 2020-04-23 at 17 02 13" src="https://user-images.githubusercontent.com/7221533/80149382-929a8980-8584-11ea-88dc-0803b954028a.png">